### DISCUSSION TITLE
为啥我的界面是这样的

### DISCUSSION THREAD

**Discussion by fat603328** [share121/inter-knot#3145] [其他]
只要我点开回复发帖或者是换头像我的界面都是这样的，怎么变成正常的样子呀
![Uploading Screenshot_2024-08-19-20-42-00-08_a252b927494330cdc2c8ba3b3f952e5e.jpg…]()

**Comment by Nie7bai** [comment]
图片没有传上来<img src="https://img-static.mihoyo.com/communityweb/upload/ffb9724864fd36eff5df36a3e7145075.png" alt="吃糖葫芦" width="75" >
